LayoutKitTests/OverlayLayoutTests.swift

```diff
@@ -95,6 +95,27 @@ class OverlayLayoutTests: XCTestCase {
     }
 
     /**
+     Tests primary layout smaller than overlay.
+     Sublayouts should have same measurements.
+     */
+    func testPrimaryLayoutSmallerThanOverlay() {
+        let primaryLayout = SizeLayout<View>(minWidth: 2, minHeight: 2)
+        let sublayout = SizeLayout<View>(minWidth: 10, minHeight: 10, alignment: .fill)
```

Comment:
Does this work fine without `fill` too? In no circumstance should the overlay appear bigger than the primary layout.